apps/api/src/app/events/usecases/send-message/send-message-chat.usecase.ts

```diff
@@ -260,6 +270,7 @@ export class SendMessageChat extends SendMessageBase {
 
   private async sendMessage(
     chatWebhookUrl: string,
+    channelSpecification: string | undefined,
```

Comment:
If it can be undefined, therefore optional, the argument should be also latter than the mandatories in the argument order.
```suggestion
    channelSpecification?: string | undefined,
```

Reply:
Updated